travis_opam.ml

```diff
@@ -0,0 +1,150 @@
+(*
+ * Copyright (c) 2015 David Sheets <sheets@alum.mit.edu>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ *)
+
+(* First thing's first: conjure tools. *)
+
+open Yorick
+
+(* User-defined variables *)
+
+(* The package name *)
+let pkg = getenv_default "PACKAGE" "my-package"
+
+(* Extra remotes to stack on top of the initialization remote *)
+let extra_remotes = list (getenv_default "EXTRA_REMOTES" "")
+
+(* Any extra pins to use *)
+let pins = list (getenv_default "PINS" "")
+
+(* Run the basic installation step *)
+let install_run = bool_of_string (getenv_default "INSTALL" "true")
```

Comment:
the type of this was a string that was defaulted to `true` and tested against `"true"`

This should probably be `fuzzy_bool_of_string`.